src/js/components/TextInput.js

```diff
@@ -303,11 +303,12 @@ export default class TextInput extends Component {
 
   render () {
     const {
-      className, defaultValue, value, ...props
+      className, defaultValue, value, placeHolder, ...props
     } = this.props;
     delete props.suggestions;
     delete props.onDOMChange;
     delete props.onSelect;
+    delete props.placeHolder;
```

Comment:
I dont think we need to add the delete here. 

```
const {
  className, defaultValue, value, placeHolder, ...props
} = this.props;
```

This will take care of it.